packages/components/src/Headings/Headings.tsx

```diff
@@ -12,8 +12,17 @@
 
 import styled from 'styled-components'
 
-export const Heading3 = styled.div`
+// Form Heading 2
+export const Heading2 = styled.h2`
+  ${({ theme }) => theme.fonts.h2};
+  color: ${({ theme }) => theme.colors.grey600};
+  padding-bottom: 8px;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.grey200};
+`
+
+// Form Heading 3
+export const Heading3 = styled.h3`
   ${({ theme }) => theme.fonts.h3};
   color: ${({ theme }) => theme.colors.grey600};
-  margin-top: 24px;
+  padding-top: 20px;
 `
```

Comment:
Do we want to document these somewhere in the style guide?

Reply:
as these are specific to the form. I didn't think we need to